PWGCF/Flow/Tasks/flowGfwOmegaXi.cxx

```diff
@@ -1111,12 +1205,27 @@ struct FlowGfwOmegaXi {
               ((std::fabs(itsResponse.nSigmaITS<o2::track::PID::Kaon>(bachelor)) < cfgNSigma[8]) || bachelor.pt() < lowpt) && ((std::fabs(itsResponse.nSigmaITS<o2::track::PID::Proton>(posdau)) < cfgNSigma[7]) || posdau.pt() < lowpt) && ((std::fabs(itsResponse.nSigmaITS<o2::track::PID::Pion>(negdau)) < cfgNSigma[6]) || negdau.pt() < lowpt)) {
             registry.fill(HIST("InvMassOmega_all"), casc.pt(), casc.mOmega(), casc.eta(), cent);
             isOmega = true;
+
+            setCurrentParticleWeights(weff, wacc, bachelor, vtxz, 0);
+            fGFW->Fill(bachelor.eta(), 1, bachelor.phi(), wacc * weff * wloc, 4096);
+            setCurrentParticleWeights(weff, wacc, posdau, vtxz, 0);
+            fGFW->Fill(posdau.eta(), 1, posdau.phi(), wacc * weff * wloc, 4096);
+            setCurrentParticleWeights(weff, wacc, negdau, vtxz, 0);
+            fGFW->Fill(negdau.eta(), 1, negdau.phi(), wacc * weff * wloc, 4096);
```

Comment:
The same applies in here for each of the four cases
Consider incorporating the replicated code in a local lambda function if feasible